go.sum

```diff
@@ -812,8 +812,8 @@ github.com/vishvananda/netns v0.0.0-20171111001504-be1fbeda1936/go.mod h1:ZjcWmF
 github.com/vmware-tanzu/vm-operator-api v0.1.3 h1:4vxewu0jAN3fSoCBI6FhjmRGJ7ci0R2WNu/I6hacTYs=
 github.com/vmware-tanzu/vm-operator-api v0.1.3/go.mod h1:mubK0QMyaA2TbeAmGsu2GVfiqDFppNUAUqoMPoKFgzM=
 github.com/vmware/govmomi v0.20.3/go.mod h1:URlwyTFZX72RmxtxuaFL2Uj3fD1JTvZdx59bHWk6aFU=
```

Comment:
I think we can remove older references in go.sum file.
just make sure to execute `go mod tidy` to ensure everything is correct.

Reply:
I ran go mod tidy, this was the result. These lines didnt get removed ..

Reply:
Should i remove them anyway?

Reply:
if go mod tidy adds back this line, leave it, otherwise, this will appear as diff every time we make further change.

Reply:
After removing the line:
```
$ go mod tidy
$ git diff
$
```
It gets added back

Reply:
ok then keep as it is